### ISSUE TITLE
Special chars in API JSON

### ISSUE THREAD

**Issue by gryphon** [ossrs/srs#1757]
**描述(Description)**

HTTP API does not encode special chars in json, for example "\" in stream name.

> 描述你遇到了什么问题(Please description your issue here)

1. SRS版本(Version): `v3.0-b4`

**重现(Replay)**

> 重现Bug的步骤(How to replay bug?)

Just start stream with name "\\\\\" and it will then produce invalid JSON in /v1/streams API.

**期望行为(Expect)**

> 描述你期望发生的事情(Please describe your expectation)

JSON could be parseable even if it contains invalid stream names

**Comment by winlinvip** [comment]
I will consider it.

`TRANS_BY_GPT3`

**Comment by winlinvip** [comment]
Example of returning JSON in Node.js:

```
'use strict';

const Koa = require('koa');
const Router = require('koa-router');

const moment = require('moment');

const app = new Koa();

const router = new Router();
router.all('/hello-koa', async (ctx) => {
  ctx.body = {
    errCode: 0,
    errorMessage: "",
    data: {
`msg: 'Hello视频',`
      // @see https://momentjs.com/docs/#/manipulating/utc-offset/
      now: moment().utcOffset(moment.duration(8, 'hours').asMinutes()).format('YYYY-MM-DD HH:mm:ss'),
    },
  }
});
app.use(router.routes());

app.listen(9000, () => {
  console.log(`Server start on http://localhost:9000`);
});
```

![image](https://user-images.githubusercontent.com/2777660/138008472-33821a3f-7524-4ba7-99c3-e2136738524f.png)

[nodejs-json-utf8.pcapng.zip](https://github.com/ossrs/srs/files/7377315/nodejs-json-utf8.pcapng.zip)

`TRANS_BY_GPT3`

**Comment by winlinvip** [comment]
Fixed by 34b968aab
